### PR TITLE
Update multi-node README to check ssh connection issue

### PR DESCRIPTION
**Pull request by yeonsily**
# What does this PR do?

Fixes # (issue)

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you make sure to update the documentation with your changes?
- [ ] Did you write any new necessary tests?

### CODE REVIEW

File: examples/multi-node-training/README.md [hunk 1]
```diff
@@ -67,6 +67,28 @@ Finally, on each system, add all hosts (including itself) to `known_hosts`. The
    ssh-keyscan -p 3022 -H 10.10.100.104 >> ~/.ssh/known_hosts
    ```
 
+You can check if ssh port is working fine,
```
Comment (libinta) [inline examples/multi-node-training/README.md:70]: You can check if ssh port is working with the following commands:

File: examples/multi-node-training/README.md [hunk 2]
```diff
@@ -67,6 +67,28 @@ Finally, on each system, add all hosts (including itself) to `known_hosts`. The
...
+
+a. Run "lsof -i" inside both dockers. It should be something like below.
```
Comment (libinta) [inline examples/multi-node-training/README.md:72]: should we change this to 1,2,3,4?

File: examples/multi-node-training/README.md [hunk 3]
```diff
@@ -67,6 +67,28 @@ Finally, on each system, add all hosts (including itself) to `known_hosts`. The
...
+sshd     35 root    4u  IPv6 23262523      0t0  TCP *:3022 (LISTEN)
+```
+If no sshd, then do the following to restart sshd.
+```bash
+sed -i 's/#Port 22/Port 3022/g' /etc/ssh/sshd_config
+sed -i 's/#   Port 22/    Port 3022/g' /etc/ssh/ssh_config
+sed -i 's/#PermitRootLogin prohibit-password/PermitRootLogin yes/' /etc/ssh/sshd_config
+service ssh restart
+```
+b. Try ssh connect "ssh -p 3022  IP-address" to each other to make sure the nodes can talk to each other.
```
Comment (libinta) [inline examples/multi-node-training/README.md:85]: Test ssh with command "ssh -p 3022 IP-address" to each other to make sure the nodes can communicate with each other.

File: examples/multi-node-training/README.md [hunk 4]
```diff
@@ -67,6 +67,28 @@ Finally, on each system, add all hosts (including itself) to `known_hosts`. The
...
+
+c. Try gaudi_spawn.py with world_size 8 for few steps if it can run on the docker on each node.
```
Comment (libinta) [inline examples/multi-node-training/README.md:87]: Try gaudi_spawn.py training command with world_size 8 for few steps to make sure the command works for 8 ranks.

File: examples/multi-node-training/README.md [hunk 5]
```diff
@@ -67,6 +67,28 @@ Finally, on each system, add all hosts (including itself) to `known_hosts`. The
    ssh-keyscan -p 3022 -H 10.10.100.104 >> ~/.ssh/known_hosts
    ```
 
+You can check if ssh port is working with the following command:
+
+1. Run "lsof -i" inside docker of each node to make sure sshd is up. It should be something like below.
```
Comment (ssarkar2) [inline examples/multi-node-training/README.md:72]: "lsof -i" -> `lsof -i`

File: examples/multi-node-training/README.md [hunk 6]
```diff
@@ -67,6 +67,28 @@ Finally, on each system, add all hosts (including itself) to `known_hosts`. The
...
+sshd     35 root    4u  IPv6 23262523      0t0  TCP *:3022 (LISTEN)
+```
+If no sshd, then do the following to restart sshd.
+```bash
+sed -i 's/#Port 22/Port 3022/g' /etc/ssh/sshd_config
+sed -i 's/#   Port 22/    Port 3022/g' /etc/ssh/ssh_config
+sed -i 's/#PermitRootLogin prohibit-password/PermitRootLogin yes/' /etc/ssh/sshd_config
+service ssh restart
+```
+2. Test ssh with command "ssh -p 3022 IP-address" to each other to make sure the nodes can communicate with each other.
```
Comment (ssarkar2) [inline examples/multi-node-training/README.md:85]: `ssh -p 3022 IP-address`

**Review by regisss — APPROVED** [review]

**Pull request merged**